flocker/node/_docker.py

```diff
@@ -554,6 +554,20 @@ def _list():
                     name = name[1 + len(self.namespace):]
                 else:
                     continue
+                # Retrieve environment variables for this container,
+                # disregarding the HOME and PATH environment variables that are
+                # included in every container and do not translate back to the
+                # deployed Flocker applications.
+                unit_environment = []
+                environment_data = data[u"Config"][u"Env"]
+                for environment in environment_data:
+                    if not environment.startswith(("HOME", "PATH")):
```

Comment:
Or actually , you probably want `("HOME=", "PATH=")` so you don't skip things like `"HOME_BLAH"` by mistake.